backend/index.ts

```diff
@@ -35,3 +36,53 @@ const port = 8000;
 app.listen(8000, () => {
     console.log(`Server started: http://localhost:${port}/`);
 }); 
+
+
+
+// Configure Strategy for Login
+
+// user is prompted to sign in with their username and password by rendering a form
+// defining a route
+
+app.get('/login/password',
+  function(req, res, next) {
+    res.render('login');
+  });
+
+app.post('/login/password',
+  passport.authenticate('local', { failureRedirect: '/login', failureMessage: true }),
+  function(req, res) {
+    console.log(req.body)
+    res.status(200).send("we logged in")
+  });
+
+var passport = require('passport');
+var LocalStrategy = require('passport-local');
+var crypto = require('crypto');
+var mysql = require('mysql12')
+
+// connecting to the database
+var connection = mysql.createConnection({
```

Comment:
this will be caught by our linter eventually but modern js uses `let` and `const` and not `var` anymore. let's also convert these into `import` statements instead of `require`'s

Reply:
Okay good to know will import these instead.